UPGRADING.md

```diff
@@ -16,95 +16,50 @@ openseach-php removes the hard-coded dependency on the [Guzzle HTTP client](http
 
 You can continue to use Guzzle, but will need to configure it as a PSR-18 HTTP Client.
 
-### HTTP Client Auto-Discovery
+### PSR-18 HTTP Client Interfaces
```

Comment:
The TOC needed an update, did in https://github.com/opensearch-project/opensearch-php/pull/290.